tests/utils/test_calc.cairo

```diff
@@ -90,10 +110,13 @@ fn given_normal_conditions_when_sum_return_uint_128_then_works() {
         sum_return_uint_128(BoundedInt::max(), i128_new(1, true)) == BoundedInt::max() - 1,
         'Should be max - 1'
     );
-    // assert(
-    //     sum_return_uint_128(BoundedInt::max(), min_i128() + i128_new(1, false)) == max_i128_as_u128() + 1,
-    //     'Should be max/2 +1 (1)'
-    // );
+    assert(
+        sum_return_uint_128(
+            BoundedInt::max(), min_i128() + i128_new(1, false)
+        ) == max_i128_as_u128()
+            + 2,
```

Comment:
Getting +1 with gmx but this might be related to the issue we discussed with custom implementation